docs/tutorials/adding-a-module-to-your-runtime.md

```diff
@@ -199,7 +201,7 @@ pub use contracts::Gas;
 /* --snip-- */
 ```
 
-If you have followed our [other basic tutorials](tutorials/), you may remember that we need to implement a `contracts::Trait` and also add `Contracts: contracts,` to our `construct_runtime!` macro.
+If you have followed our [other basic tutorials](/tutorials), you may remember that we need to implement a `contracts::Trait` and also add `Contracts: contracts,` to our `construct_runtime!` macro.
```

Comment:
@JoshOrndorff for some reason I had to revert it to your proposal. It was not working in any other way :confused: